examples/perm_DC_omega.py

```diff
@@ -0,0 +1,54 @@
+from examples.agents.simple_controllers import Controller
+import time
+import sys, os
+
+sys.path.append(os.path.abspath(os.path.join('..')))
+import gym_electric_motor as gem
+from gym_electric_motor import reference_generators as rg
+from gym_electric_motor.visualization import MotorDashboard
+
+"""
+Run this file from within the 'examples' folder:
+>> cd examples
+>> python perm_DC_omega.py
+
+Description:
+        Environment to control a  permanently excited DC motor.
+        Controlled Quantity: 'omega'
+        Limitations: Physical limitations of the motor will be Current.
+        Converter : FourQuadrantConverter from converters.py
+"""
+
+if __name__ == '__main__':
+
+    """
+       Continuous mode: The action is the average (normalized) voltage per time step which is assumed to be transferred to a PWM/SVM 
+                        converter to generate the switching sequence.
+
+       Discrete mode: The action is the switching state of the power converter i.e., a quantity from a discrete set of
+                      switching states which can be generated by the converter.     
+    """
+    env = gem.make(
+        'DcPermExDisc-v1',
+        visualization=MotorDashboard(plots=['omega', 'torque', 'i', 'u', 'u_sup'], visu_period=1),
+        ode_solver='scipy.solve_ivp', solver_kwargs=dict(),
+        reference_generator=rg.SwitchedReferenceGenerator(
+            sub_generators=[
+                rg.SinusoidalReferenceGenerator, rg.WienerProcessReferenceGenerator(), rg.StepReferenceGenerator()
+            ], p=[0.1, 0.8, 0.1], super_episode_length=(1000, 10000)
+        )
+    )
```

Comment:
The visualization is not well configured: the example time step size is very small due to the discrete action space, i.e., it really takes some time to simulate just a few milliseconds. However, the standard visualization window by default shows a time intervall of 3s -> for most of the time the output window is mostly blank and only a small fraction of the let figure part is used. 

Please reconfigure the visualization options such that the depicted time intervall is more suitable to the given control problem.

Reply:
@praneeth-b : Can you assist here? 
How can Rohith change the default time interval depicted when a visualization window is opened? Currently it seems to be 3 s by default, which would be too long for the given example.

Reply:
The default time interval for the visualization window in the latest nightly is 1s and not 3s.
The value of 1s is because of the 'x_width' variable which is hard-coded to 1s and not parametrizable by the user at the moment. 
So the visualization is basically a sliding window of the latest 1s of the simulation and the number of points plotted is  x_width / tau = 1s / tau.
I tested  Rohit's code on his branch by setting a tau = 1e-4 and I do not see any problem in the visualization (image attached). @RohithCharanD : could you please elaborate what exactly was the problem with the visualization.?
![Screenshot from 2020-10-20 21-35-38](https://user-images.githubusercontent.com/62220045/96636172-30e18f00-131d-11eb-8a84-ed33ab542455.png)

Reply:
mhh, I thought that the time length depicted in a window is parametrizable. Not ideal with that fixed one second length, e.g. if you want to debug and only require a couple of milliseconds to be displayed. 

Also tau will be smaller for this and similar examplars since the action space is discrete and looking from the application point of view tau = 1e-4 would be too large then. Hence, it will take some time until that 1 s window is filled with some 'signal live'.

Reply:
@RohithCharanD : Based on the feedback of @praneeth-b  I believe we cannot resolve this issue directly within this example. 

Did you implent the other change requests so far?